### DISCUSSION TITLE
Can not log in when user open link on messenger facebook app

### DISCUSSION THREAD

**Discussion by mbvb1223** [abacritt/angularx-social-login#383] [Q&A]
I install this package and it works well on Chrome browser but if User open a link from messager or Facebook app then can not log in with Google and Facebook.
With Google, after login successfully then it shows loading and can not close/redirect to my website
With Facebook, after login successfully then it redirect to my website but User does not login.

Does anyone have the same issue or any advice for me?

https://abacritt.github.io/angularx-social-login/

**Comment by jaibatrik** [comment]
Have to see on how to make redirects work. Haven't tried it yet.

Try to have the `autoLogin` value as `true` in the configuration.

**Reply by mbvb1223** [reply]
I tried to with **"ux_mode": redirect** for google provider but User can not log in also.
I see some guys raise this issue but it seem we did not find any solution for it :(
